src/components/va-alert/va-alert.tsx

```diff
@@ -108,14 +108,15 @@ export class VaAlert {
   render() {
     const { backgroundOnly, status, visible, closeable } = this;
     const classes = `alert ${status} ${backgroundOnly ? 'bg-only' : ''}`;
+    const role = status === 'error' ? 'alert' : null;
+    const ariaLive = status === 'error' ? 'assertive' : null;
 
     if (!visible) return <div aria-live="polite" />;
     
     return (
       <Host>
-        <div class={classes}>
+        <div role={role} aria-live={ariaLive} class={classes}>
           <i aria-hidden="true" role="img"></i>
-          <span class="sr-only">Alert: </span>
```

Comment:
In now announces as an alert since it has the correct role in place. Removing this to prevent double announcing the "Alert" text

Reply:
I like the way you're adding role and aria-live attributes when needed, ignoring when not.